browser/src/ClinvarVariantsTrack/ClinvarVariantPropType.ts

```diff
@@ -1,6 +1,33 @@
 import PropTypes from 'prop-types'
 
-const ClinvarVariantPropType = PropTypes.shape({
+type ClinvarVariantPropType = {
```

Comment:
I love that this tool generated these types so nicely. And kudos to @nawatts for disciplined use of PropTypes that enabled this.

Reply:
My only beef with PropTypes really is that they're a half measure, in a hypothetical world where I couldn't have types checked at compile time on everything, types checked at runtime on props only would be better than nothing.